### PR TITLE
Get global namespace

### PR DESCRIPTION
**Pull request by teresaromero**
## Proposed changes

- Add to the Token the globalnamespace value in order to be able to push images to the global repository

### CODE REVIEW

File: pkg/okteto/auth.go
Comment (pchico83) [inline pkg/okteto/auth.go:148]: If you run this against Cloud, it will fail, because we don't support this field yet.
We should catch the error in that case, and do the query without `globalnamespace`.
There is similar code when deploying a pipeline, for example.

**Review by pchico83 — APPROVED** [review]

File: pkg/okteto/auth.go
Comment (ifbyol) [inline pkg/okteto/auth.go:154]: Maybe it is out of the scope of this PR, but maybe we can start using IntrospectionQueries to this kind of things: https://graphql.org/learn/introspection/.

Executing this query against my dev cluster

```
query IntrospectionQuery {
  __type(name: "me"){
    name
    fields {
      name
    }
  }
}
``` 

I got

```
{
  "data": {
    "__type": {
      "fields": [
        {
          "name": "avatar"
        },
        {
          "name": "buildkit"
        },
        {
          "name": "capabilities"
        },
        {
          "name": "certificate"
        },
        {
          "name": "config"
        },
        {
          "name": "email"
        },
        {
          "name": "externalID"
        },
        {
          "name": "gitSSHSupport"
        },
        {
          "name": "githubID"
        },
        {
          "name": "id"
        },
        {
          "name": "integrations"
        },
        {
          "name": "name"
        },
        {
          "name": "namespace"
        },
        {
          "name": "new"
        },
        {
          "name": "onboarded"
        },
        {
          "name": "personalAccessTokens"
        },
        {
          "name": "plan"
        },
        {
          "name": "planSubscribed"
        },
        {
          "name": "privateEndpointToken"
        },
        {
          "name": "quickstarts"
        },
        {
          "name": "quotaPlan"
        },
        {
          "name": "registry"
        },
        {
          "name": "secrets"
        },
        {
          "name": "super"
        },
        {
          "name": "token"
        },
        {
          "name": "trialExpiration"
        },
        {
          "name": "userDefinedNamespaces"
        }
      ],
      "name": "me"
    }
  }
}
```

I think this is most robust since it is based on the graphql schema defined on the server

Comment (teresaromero) [reply pkg/okteto/auth.go:154]: Should we open a ticket for this?

Comment (pchico83) [reply pkg/okteto/auth.go:154]: Yeap, I would implement it right after this is merge

Comment (ifbyol) [reply pkg/okteto/auth.go:154]: Yes, we can merge this PR as it is now and include the query instrospection later if we see the value it could provide, but do not block this PR for that

**Review by ifbyol — APPROVED** [review]

**Pull request merged**
